peers/cache_test.go

```diff
@@ -50,19 +64,19 @@ func TestMultipleTopics(t *testing.T) {
 	require.NoError(t, err)
 	topics := []discv5.Topic{discv5.Topic("first"), discv5.Topic("second")}
 	for i := range topics {
-		peers := [3]*discv5.Node{
-			discv5.NewNode(discv5.NodeID{byte(i), 1}, net.IPv4(100, 100, 0, 3), 32311, 32311),
-			discv5.NewNode(discv5.NodeID{byte(i), 2}, net.IPv4(100, 100, 0, 4), 32311, 32311),
-			discv5.NewNode(discv5.NodeID{byte(i), 3}, net.IPv4(100, 100, 0, 2), 32311, 32311)}
-		for _, peer := range peers {
+		for j := 0; j < 3; j++ {
```

Comment:
It could be a separate function(like create N discv5 nodes). There is a lot of the same fixes.